### ISSUE TITLE
Can't render symbols

### ISSUE THREAD

**Issue by searsaw** [relax/relax#418]
When I try to make a symbol, an error saying `relax.renderChildren` isn't a function. I think this is related to the new rendering process that was added recently.

**Comment by bruno12mota** [comment]
Yes, that needs to be fixed. Shouldn't be hard, it needs to pass the symbol context to the render children function.
